config.yaml

```diff
@@ -30,3 +31,102 @@ crs:
   geo_crs: EPSG:4326  # general geographic projection, not used for metric measures. "EPSG:4326" is the standard used by OSM and google maps
   distance_crs: EPSG:3857  # projection for distance measurements only. Possible recommended values are "EPSG:3857" (used by OSM and Google Maps)
   area_crs: ESRI:54009  # projection for area measurements only. Possible recommended values are Global Mollweide "ESRI:54009"
+
+
+augmented_line_connection:
```

Comment:
A large fraction of these inputs are not needed in the microgrids version

Reply:
I cancelled the whole "augmented_line_connection" paragraph because it seemed not to be useful in the current version. I will re-add the necessary parts when needed.